client/src/org/moparforia/client/Launcher.java

```diff
@@ -67,16 +67,30 @@ public static void main(String[] args) throws Exception {
         for(int i = 1; i < m.groupCount(); i++)
             System.out.println(m.group(i));
         */
-
-        String server;
-        if (args.length > 0) {
-            server = args[0];
-        } else {
-            //server = "localhost";
-            server = "127.0.0.1";
-        }
-
-        instance = new Launcher(server);
+		
+			//easily editable optional client launch options, at the moment supports custom server ip and language pack
+			//example: client.jar -server 125.456.789.123 -lang en_US
+			
+			String server = "127.0.0.1"; //dafault server ip
+			String lang = "en_US"; //dafault language package (comes with en_US, fi_FI and sv_SE)
+			
+			if(args.length==1){ //support for "legacy" .bats where only ip provided after jar
+				server=args[0];
+			}
+			
+			for(int i = 0;i<args.length ; i++)
```

Comment:
Some error checking would be useful here.

- Out of bounds possible if there is a missing value for an argument
- check if lang has a valid format and maybe give an error message when there is not localization for a given country code
- Optional: server should bei either localhost, an IPv4 Adress or a Domain/Hostname

Reply:
I have made sure out of bounds shouldn't be possible now... But haven't implemented any kind of format checking at least yet.